src/components/blocks/download-button/_download-button.scss

```diff
@@ -0,0 +1,268 @@
+.js-modal-action.b-icon-link:hover .b-icon-link__container::after {
+    opacity: 1;
+}
+
+.b-icon-link__container.image-button__icon-link-container--active.js-modal-action::after {
+	background-color: #04d1b3 !important;
+	border-color: #04d1b3 !important;
+    opacity: 1;
+}
+
+.license-button {
+	&__button {
```

Comment:
name-spacing e.g. - either use **download-button** or **license-button**, but not both.

Reply:
renamed/removed